docs/reference/query-dsl/geo-shape-query.asciidoc

```diff
@@ -31,7 +31,7 @@ PUT /example
 {
     "mappings": {
         "properties": {
-            "location": {
+            "geo": {
```

Comment:
is it needed this change?

Reply:
no, I can roll that back. I made the change to reflect that a shape does not necessarily have to be a location